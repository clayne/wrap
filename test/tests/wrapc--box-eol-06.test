wrapc | /dev/null | | wrapc-box-06.sh | 0
