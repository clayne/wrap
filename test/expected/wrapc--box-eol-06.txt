  # This is a comment that has a '#' in it. The '#' in the middle shouldn't   #
  # be chopped off prior to wrapping.                                         #
